generation/realism_diversity_metrics/realism_diversity_metrics.ipynb

```diff
@@ -0,0 +1,841 @@
+{
```

Comment:
<p>Please help update the description about "RadImageNet" here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1771/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
thanks, I removed the `notes` but keep the reference (since we still mentions this network)